include/wifi.h

```diff
@@ -0,0 +1,59 @@
+#ifndef __WIFI_H__
+#define __WIFI_H__
+
+// Include custom debug macros
```

Comment:
This should be updated to:
`// Include custom debug macros and compile flags`

Reply:
addressed in: https://github.com/jCallon/squirt/pull/1/commits/edb88a636d2cf54a42f970d190cd054c04993412